src/EditorFeatures/Core.Wpf/QuickInfo/LazyToolTip.cs

```diff
@@ -29,11 +31,13 @@ private class LazyToolTip : ForegroundThreadAffinitizedObject
                 IThreadingContext threadingContext,
                 FrameworkElement element,
                 Func<DisposableToolTip> createToolTip)
-                : base(threadingContext, assertIsForeground: true)
```

Comment:
any usage of `assertIsForeground: true` (not hte default) had an assert directly added.